azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java

```diff
@@ -45,6 +45,7 @@
   private JavaProcessJob job = null;
   private Props props = null;
   private Logger log = Logger.getLogger(JavaProcessJob.class);
+  private AllJobExecutorTests jobExecutorTests = null;
```

Comment:
What's the benefit of having jobExecutorTests as a member variable? 
What do you think about making the method static? Afterall, it is a stateless method.